api/src/main/java/org/infinispan/api/collections/reactive/KeyValueStoreConfig.java

```diff
@@ -0,0 +1,65 @@
+package org.infinispan.api.collections.reactive;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.infinispan.api.marshalling.Marshaller;
+
+/**
+ * Configure the key value store
+ *
+ * @since 10.0
+ */
+public class KeyValueStoreConfig {
```

Comment:
`KeyValueStoreConfig` should be interface with an public+final implementation.

Reply:
ok